### ISSUE TITLE
NVDA does not read the subtitles or alerts in youtube while using microsoft edge

### ISSUE THREAD

**Issue by Fatihyuksek** [nvaccess/nvda#8337]
### Steps to reproduce:
> A list of the steps you take to demonstrate the problem. 
>
> Example:
>
> 1. Open Chrome
> 2. Browse to www.google.com
> 3. Type "Hello"
> 4. Notice an error sound when enter is pressed.

> Please also remember to attach a zip of any files required to reproduce the issue.

### Expected behavior:
> Tell us what should happen.

### Actual behavior:
> Tell us what happens instead.

### System configuration:
NVDA version:
> Example: next-14027,c80e529f

NVDA Installed or portable:

Other information:
> Example: Running in a VM

Windows version:
> Example: Windows 10 Version 1607 Build 14393.1066

Name and version of other software in use when reproducing the issue:

### Other questions:

Does the issue still occur after restarting your PC?

Have you tried any other versions of NVDA?
> Please list them and the result
NVDA des not read the alerts like add is displayed, add can now be skipped or subtitles in videos on Youtube with Microsoft edge

**Comment by josephsl** [comment]
Hi, which Windows 10 release is this? Thanks. CC @MichaelDCuirran

**Comment by Fatihyuksek** [comment]
İt is the latest release. This bug was present in older releases as well

iPhone’umdan gönderildi

Joseph Lee <notifications@github.com<mailto:notifications@github.com>> şunları yazdı (28 May 2018 20:19):

Hi, which Windows 10 release is this? Thanks. CC @MichaelDCuirran

**Comment by josephsl** [comment]
Hi, confirmed. Looks like recent YouTube redesign may have been responsible for that, as it didn’t occur in old builds. Thanks.

**Comment by josephsl** [comment]
Hi,

Ah yes, let's see if this is still reproducible in NVDA 2019.2 RC1, and if yes, we need to keep this open; I'm thinking it can be closed as this is due to YouTube rather than EdgeHTML.

Thansk.

**Comment by Adriani90** [comment]
This is reproducible in every browser with NVDA. I didn't find any browser which let's NVDA to read the captions and subtitles automatically. Very strange.

You can test for example with this video:
https://www.youtube.com/watch?v=QRS8MkLhQmM

But maybe I am doing something wrong here. I tried in full screen as well, I placed the mouse on the captions or subtitles, no chance.

**Comment by Adriani90** [comment]
cc: @derekriemer, @Qchristensen.

**Comment by feerrenrut** [comment]
I couldn't get this working well at all. What I could do:
- Enable closed captions. Tab through the controls for subtitles / closed captions. It is after volume and before settings.
- I could read the current caption in browse mode. Wait until somebody is speaking in the video, press pause, then in browse mode press up arrow until you read the captions.

Looking at the implementation, the captions are in an `aria-live=assertive` container.

**Comment by Adriani90** [comment]
in browse mode I can read them too. I think however it would more confortable having them read automatically while watching the video. I recall that there was a PR in place which refactors the live region handler but I am not sure if it will fix this issue.

Von meinem iPhone gesendet

**Comment by Adriani90** [comment]
It is the PR #9079.

**Comment by feerrenrut** [comment]
The goal of PR #9079 is to expose the politeness setting for aria-live. NVDA should already be announcing aria-live changes. A minimal test case would help us to reproduce the issue. I believe with this youtube example the problem can be reproduced in Firefox and Chrome also.

**Comment by cary-rowen** [comment]
Any progress on this so far?
BTW, Why not fill out the template?

**Comment by Adriani90** [comment]
@leonardder, @jcsteh can this be considered fixed as per #15331?

**Comment by cary-rowen** [comment]
@Adriani90 
No, I don't think this has anything to do with #15331!

thanks.

**Comment by jcsteh** [comment]
These could be read by auto refresh OCR, sure, but that's not ideal due to accuracy, etc.
